backend/wmg/cube_creation/load_cube.py

```diff
@@ -174,8 +175,8 @@ def enumerate_timestamps_and_objects(objects):
             object.delete()
 
 
-def upload_cube_to_s3(group_name, timestamp):
-    sync_command = ["aws", "s3", "sync", f"{group_name}/cube", f"{get_wmg_bucket_path()}/{timestamp}/cube"]
+def upload_cube_to_s3(group_name, suffix, timestamp):
+    sync_command = ["aws", "s3", "sync", f"{group_name}/{suffix}", f"{get_wmg_bucket_path()}/{timestamp}/{suffix}"]
```

Comment:
can't we just sync the entire local parent dir to s3, rather than individually for each cube?

if we stick with this usage, note that `group` is a little confusing, since [TileDB also uses "group" term](https://docs.tiledb.com/main/how-to/groups).  I don't think we're using it in that sense, right?  

I think a signature like this would be simpler: `sync_dir_to_s3(local_dir, remote_s3_path)`. And then let the caller append the `suffix` to both args being passed.